.github/rt-dictionary.yaml

```diff
@@ -0,0 +1,8 @@
+# This file is a dictionary used by the [link-dependencies](https://github.com/newrelic/release-toolkit/tree/main/link-dependencies) action.
+# Notice that the implementation uses dep.To.ToString that removes the leading v if present.
+dictionary:
+  # TODO: Unable to link dependency for Power DNS because it uses a weird changelog URL scheme:
```

Comment:
This might be worth raising upstream. The code is already [using templates](https://github.com/newrelic/release-toolkit/blob/aaee8b2c7af1ad6115c1479b23ea6da6fd94057b/src/changelog/linker/mapper/dictionary.go#L74), and is already importing the semver package, so it should be a matter of a couple lines of code to expose the `major` and `minor` functions to the template, such that you can do something like:
```
https://doc.powerdns.com/authoritative/changelog/{{.To | major}}.{{.To | minor}}.html#change-{{.To}}
```

Hopefully we can find a rt maintainer that can listen to this request 🤞🏻

Reply:
I cannot open an issue from my personal laptop because i am redirected to my company's Okta.

I'll open a ticket tomorrow morning.

Reply:
newrelic/release-toolkit#200